src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py

```diff
@@ -114,6 +114,7 @@ def load_arguments(self, _):
         c.argument('zone_resilient', min_api='2017-12-01', arg_type=get_three_state_flag(), help='Specifies whether an image is zone resilient or not. '
                    'Default is false. Zone resilient images can be created only in regions that provide Zone Redundant Storage')
         c.argument('storage_sku', arg_type=disk_sku, help='The SKU of the storage account with which to create the VM image. Unused if source VM is specified.')
+        c.argument('os_disk_caching', arg_type=get_enum_type(CachingTypes), help="Storage caching type for the image's OS disk. Default: None")
```

Comment:
You likely don't need the `Default: None` in the help. Most optional parameters will default to a null value.

Reply:
@williexu, I mentioned the default value used to keep in line with the os-disk-caching parameter's help text and behavior in [`VM create`](https://github.com/Azure/azure-cli/blob/dev/src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py#L439). The help text specifies that the default os disk caching is "ReadWrite".

One of the os caching enum's values is the string "None" (not null). I think it would be good to let users know what default os_disk_caching is used to create an image if the option is not used, even if in this case it is "None".  I will update the code to explicitly use the enum's "None" value, if os caching is unspecified.